### DISCUSSION TITLE
Coolify 4.0 is not supported Weblate?

### DISCUSSION THREAD

**Discussion by dungsil** [coollabsio/coolify#1437] [Q&A] [answered]
hello
There is a statement that the Weblate service was added in version 3.9.0 #582 , but I can't find it. Is it still not supported in version 4.0?

**Comment by andrasbacsai** [comment, marked as answer]
I've added it, just for you (and for others, ofc) 😁

**Reply by dungsil** [reply]
Thanks for your kind help 👍

**Reply by dungsil** [reply]
hello
When configuring Weblate, I get an authentication error and it won't run. I (probably) need to set a password, is it possible to specify it?

`redis.exceptions.AuthenticationError: Authentication required.`

**Reply by andrasbacsai** [reply]
I take a look!

**Reply by andrasbacsai** [reply]
It should be fixed now.

**Reply by dungsil** [reply]
Thanks!
